src/ai/susi/server/api/aaa/GetUsers.java

```diff
@@ -148,7 +148,7 @@ public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Aut
                   }
 
                   if(accounting.getJSON().has("signupTime")) {
-                    String signupTime = accounting.getJSON().getString("signupTime").substring(0, 8);
+                    String signupTime = accounting.getJSON().getString("signupTime").substring(8,16);
```

Comment:
This is different when you run locally and on actually instantiated server. Undo this

Reply:
@fragm3 Done